### PR TITLE
[bitnami/keycloak] HTTPS between the ingress and the pods

### PR DESCRIPTION
**Pull request by yesteph**
**Description of the change**

Allow the ingress to talk the pods via HTTPS port.

**Benefits**

Not plain HTTP between the Ingress and the Keycloak pods.
It is relevant in contexts without NetworkPolicies.

**Possible drawbacks**

N/A

**Applicable issues**

N/A

**Additional information**

N/A

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/keycloak/Chart.yaml [hunk 1]
```diff
@@ -26,4 +26,4 @@ name: keycloak
 sources:
   - https://github.com/bitnami/bitnami-docker-keycloak
   - https://github.com/keycloak/keycloak
-version: 1.4.3
+version: 2.1.0
```
Comment (Mauraza) [inline bitnami/keycloak/Chart.yaml:29]: the actual version is 2.0.1, could you check for 2.0.2?

**Review by Mauraza — COMMENTED** [review]
Hi @yesteph,

thanks for de contribution!! Could you check the small comment?

File: bitnami/keycloak/Chart.yaml (hunk 1 above)
Comment (yesteph) [reply bitnami/keycloak/Chart.yaml:29]: Hello @Mauraza,

You are right, actual is 2.0.1.
My MR relates to an improvement not a bugfix. That's the reason I propose 2.1.0.

Tell me.

Comment (Mauraza) [reply bitnami/keycloak/Chart.yaml:29]: right, we leave it as a minor

**Comment by Mauraza** [comment]
LGTM

**Pull request merged**